src/context/reducer/products-reducer.jsx

```diff
@@ -7,6 +7,34 @@ const productsReducer = (state, action) => {
       };
     }
 
+    case 'RATING_RANGE': {
+      return {
+        ...state,
+        ratingRange: action.payload.ratingRange,
+      };
+    }
+
+    case 'CATEGORY_CHANGE': {
+      const payloadCategory = action.payload.categoryName;
+
+      if (state.categoryName.includes(payloadCategory)) {
+        const newCategoryNames = state.categoryName.filter(
+          (category) => category != payloadCategory
```

Comment:
Is there a reason for not using the strict operator? You have used the strict inequality operator `!==` in other places.

Reply:
No reason for it, it was a typo, changed it back to strict inequality.